http/src/main/resources/init-resources/cloud-init.yml

```diff
@@ -0,0 +1,6 @@
+#cloud-config
+
+bootcmd:
+- fsck.ext4 -tvy /dev/sdb
+- mkdir -p /mnt/disks/work
+- mount -t ext4 -O ... /dev/sdb /mnt/disks/work
```

Comment:
Is this `…` supposed to be there?

Reply:
yea...I'm not sure exactly what it is...but it's from [this doc](https://cloud.google.com/container-optimized-os/docs/concepts/disks-and-filesystem#mounting_and_formatting_disks)

Reply:
I feel like that's a placeholder for `mount` option list…

> Refer to the Linux documentation for a full set of options supported by the fsck.ext4 and mount commands.

https://man7.org/linux/man-pages/man8/mount.8.html

Previously we used `-o discard,defaults`: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/gce-init.sh#L203

Reply:
should I change it to `discard.defaults` instead? think I did test out to see if PD works